.taskcluster.yml

```diff
@@ -89,9 +89,9 @@ tasks:
       - name: taskcluster-client-web
         image: 'taskcluster/browser-test:${node}'
         command: |
+            yarn lint clients/client-web &&
```

Comment:
It looks like this is already included in the `lint` task by virtue of the changes you made in the top-level `package.json`.  In fact, the comand here just adds `clients/client-web` to the end of that command-line, which I think will just result in the linter scanning that directory twice.  So, this can probably be omitted..